### PR TITLE
docs: Update UG for Add command

### PR DESCRIPTION
**Pull request by nickczh**
# Checklist for feature implementation and modification
Closes #<Issue_Number>
- [ ] Added relevant milestone (under milestone tab on the right)
- Implementation
  - [ ] Sources (if any) credited under both relevant section(s) and Developer Guide (Acknowledgement)
  - [ ] JUnit Test(s)
- Documentation
  - [ ] JavaDoc for classes and methods
  - Developer Guide
    - [ ] Design (Can add as a todo statement for later on)
    - [ ] Implementation (Can add as a todo statement for later on)
    - [ ] Relevant User Story present
    - [ ] Non-functional requirement(s) that the feature addresses (if any)
    - [ ] Glossary of the (technical) term(s) used (if any)
    - [ ] Specialised instructions for manual testing (if any)
  - User Guide
    - [ ] Description and usage of feature (if any)
  - Contributions (under `/docs/team/*.md`)
    - [ ] Code/Feature/Enhancement contribution
    - [ ] Developer Guide contribution
    - [ ] User Guide contribution
    - [ ] Misc and team-based admin tasks (if any)

# Checklist for non-implementation related PR
Closes #<Issue_Number>
- [ ] Added relevant milestone (under milestone tab on the right)
- Documentation (Contributions)
  - [ ] Developer Guide contributions (if any)
  - [ ] User Guide contributions (if any)
  - [ ] Review contributions (if any)
  - [ ] Miscellaneous and team-based admin tasks (if any)

# Additional Information

Nil

### CODE REVIEW

File: docs/UserGuide.md
Comment (wenenhoe) [inline docs/UserGuide.md:1]: Please update the table of contents to reflect the correct section

File: text-ui-test/data/dailymed/today.txt
Comment (wenenhoe) [inline text-ui-test/data/dailymed/today.txt:1]: Please remove this file

Comment (nickczh) [reply text-ui-test/data/dailymed/today.txt:1]: Resolved in 1aa0814b330cd9b3af388593e174ab142df01fdc

File: docs/UserGuide.md
Comment (nickczh) [reply docs/UserGuide.md:1]: Resolved in d75c38a141b0a4a827ae1fbf90fd6aa55254fb87

**Review by wenenhoe — APPROVED** [review]
LGTM! Thanks for making the changes

**Pull request merged**